tutorial/5_profiling/README.md

```diff
@@ -89,22 +74,34 @@ Open up the jupyter server and follow the instructions in `profiling-tensorflow.
 The profiling data will be created inside a directory `logs` in you current
 working directory.
 
-### Start TensorBoard
-Once the profiling data has been genereated we can start TensorBoard.
+
+### Connecting to TensorBoard for PyTorch and Tensorflow
+Once the profiling data has been genereated we can launch TensorBoard.
 The`tensorboard` command-line utility starts a web server listening on
 localhost, as seen below. 
-```
-tensorboard --logdir logs
-2021-02-09 11:54:44.785607: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
+```bash
+[cid@alvis1 5_profiling]$ module load tensorboard
+[cid@alvis1 5_profiling]$ tensorboard --logdir='path_to_logdir'
+
+Example of this tutorial in case using PyTorch module:
+[cid@alvis1 5_profiling]$ tensorboard --logdir='/cephyr/users/cid/Alvis/alvis-intro/tutorial/5_profiling/logs/base-tf'
+or in case using Tensorflow module:
+[cid@alvis1 5_profiling]$ tensorboard --logdir='/cephyr/users/cid/Alvis/alvis-intro/tutorial/5_profiling/logs/base.ptb'
+
 Serving TensorBoard on localhost; to expose to the network, use a proxy or pass --bind_all
-TensorBoard 2.3.0 at http://localhost:6007/ (Press CTRL+C to quit)
+TensorBoard 2.15.1 at http://localhost:6009/ (Press CTRL+C to quit)
 ```
-In the above example we need to visit `http://localhost:6007/` on the login
+
+In the above example we need to visit `http://localhost:6009/` on the login
 node - your port may be different.
 
-You will need to either use ThinLinc (see [Connecting with ThinLinc](https://www.c3se.chalmers.se/documentation/remote_graphics/))
-and connect to the Alvis logi node, or (recommended) setup a [SSH tunnel](https://www.c3se.chalmers.se/documentation/connecting/#use-ssh-tunnel-to-access-services)
-to access the UI from your computer.
+You will need to either use ThinLinc (see 
+[Connecting with ThinLinc](https://alvis1.c3se.chalmers.se:300/))
+and connect to Web Browser from Applications in the top left corner to access the UI from your computer.
```

Comment:
Yes, as I understand in our last meeting, I thought we can explain one way of lunching tensorboard which is through Portal. I will put it back.

Reply:
The link with SSH tunnel is not working, what we should replace it with?

Reply:
> Yes, as I understand in our last meeting, I thought we can explain one way of lunching tensorboard which is through Portal. I will put it back.

Yes, my initial idea was to only keep the instructions via Open OnDemand, but as that option seems flaky right now it might be best to keep an alternative.

And then I think SSH-tunnels is more flexible than Thinlinc in the long run.

> The link with SSH tunnel is not working

Ah, yes things have moved when www was updated last week. I created an issue, there were some other links as well we'll have to fix later at some point.